### ISSUE TITLE
Breaks with 32-bit NodeJS

### ISSUE THREAD

**Issue by wildmichael** [timostamm/protobuf-ts#515]
E.g. the NodeJS v16.14.0 shipped with Visual Studio 2022 is 32 bit and returns for `os.arch()` the value `"ia32"`. This breaks in [makeReleaseName() of utils.js](https://github.com/timostamm/protobuf-ts/blob/3a5b9f982d30d3422d856176d822bed027e981e4/packages/protoc/util.js#L190), where for the `"win32"` platform only the values `"x32"` and `"x64"` are checked. According to the [NodeJS documentation](https://nodejs.org/docs/latest-v16.x/api/os.html#osarch), `os.arch()` never returns `"x32"`, only `"ia32"`.

This is trivially fixed by modifying the check:
```js
        case "win32":
            if (params.arch === "x64") {
                build = 'win64'
            } else if (params.arch === "ia32") {
                build = 'win32'
            }
            break;

```

**Comment by timostamm** [comment]
This should be fixed by https://github.com/timostamm/protobuf-ts/pull/517

**Comment by timostamm** [comment]
The fix was released in [v2.9.0](https://github.com/timostamm/protobuf-ts/releases/tag/v2.9.0)
